content/50.discussion.md

```diff
@@ -198,7 +198,69 @@ As a result, the standard practice of requiring enrollment in a clinical trial i
 
 #### Access to and Representation in Clinical Trials
 
-<!-- Add #600 here -->
+Experimental treatments are often made available to patients primarily or even exclusively through clinical trials.
+The advantage of this approach is that clinical trials are designed to collect rigorous data about the effects of a treatment on patients.
+The disadvantage, however, is that access to clinical trials is not equal among all people who suffer from a disease.
```

Comment:
```suggestion
The disadvantage is that access to clinical trials is not equal among all people who suffer from a disease.
```
word felt extra since contrast starts the sentence. not strongly tied to this change